scripts/pre_commit_linter.py

```diff
@@ -1294,13 +1409,10 @@ def _check_sorted_dependencies(self):
                     '%s  Sorted dependencies check passed' % (
                         _MESSAGE_TYPE_SUCCESS))
 
-        summary_messages.append(summary_message)
-        print ''
-        print summary_message
-        if self.verbose_mode_enabled:
-            print '----------------------------------------'
-
-        return summary_messages
+            summary_messages.append(summary_message)
+            print summary_message
+            print ''
+            return summary_messages
```

Comment:
Careful with the merge. I think you lost some of @anubhavsinha98's changes (check the Files Changed diff).

Reply:
done